src/Tank/TankHealthBar.cpp

```diff
@@ -0,0 +1,39 @@
+#include "TankHealthBar.hpp"
+
+#include <SFML/Graphics/RenderWindow.hpp>
+#include <string>
+
+#include "Files.hpp"
+
+constexpr float HEALTHBAR_Y_OFFSET = -50.f;
+constexpr unsigned int HEALTHBAR_FONT_SIZE = 10;
+
+TankHealthBar::TankHealthBar(unsigned int total_health)
+    : mTotalHealth(total_health), mHealth(total_health) {
+  mFont.loadFromFile(files::asset_path() + "DejaVuSans.ttf");
+  setup_text();
+  update_string();
+}
+
+void TankHealthBar::set_position(const sf::Vector2f& pos) {
+  mText.setPosition(pos.x, pos.y + HEALTHBAR_Y_OFFSET);
+}
+
+void TankHealthBar::set_health(unsigned int health) {
+  mHealth = health;
+  update_string();
+}
+
+void TankHealthBar::draw(sf::RenderWindow& window) { window.draw(mText); }
```

Comment:
How about draw healthbar as outlined rectangle? We did implementation of health_bar, work is in-progress, take a look:
https://github.com/wrazik/TankBotFight/pull/54/files#diff-58f939eea121e70094f647e147daac20273292ac6a80380aebfb5fa3acae64cfR127

It looks like this:
<img width="74" alt="image" src="https://github.com/wrazik/TankBotFight/assets/6102580/b6099414-6778-481a-bafd-54b911e9449d">

It would be great to combine both solutions (with text in the middle).

Reply:
Two rectangles look better than I thought :)

Results of mixing bar with text:

![Screenshot from 2023-09-16 14-04-01](https://github.com/wrazik/TankBotFight/assets/45121219/f823047f-bb70-41e0-9572-e40c1975d5d0)

Also this epic diagram that helped me with calculating the offsets for red bar origin. The calculations may be a bit complicated but it works :D. I can change it to the method from `coding_dojo_2` branch if needed.

![Screenshot from 2023-09-16 14-08-49](https://github.com/wrazik/TankBotFight/assets/45121219/5e969a57-b0b3-4ebc-a69b-e87d2badaab2)